packages/react/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -253,7 +254,9 @@ export const ThirdwebProvider = <
     ) {
       return supportedChains as Readonly<Chain[]>;
     }
-    return [...supportedChains, activeChain] as Readonly<Chain[]>;
+    return uniqueBy([activeChain, ...supportedChains], "chainId") as Readonly<
+      Chain[]
+    >;
```

Comment:
Thanks or contributing to our repo! This is indeed an issue. What do you think if we do the following to filter out the `activeChain` from the `supportedChains` array? This way we don't need to add the `uniqueBy` function and can use a tested array function.

```
return [
      ...supportedChains.filter((c) => c.chainId !== activeChain.chainId),
      activeChain,
    ] as Readonly<Chain[]>;
```

Reply:
@iketw Sure, makes sense to use simpler implementation. Will make the change